Services/CRUD/CrudManager.php

```diff
@@ -225,6 +227,29 @@ public function persist(CrudEntityInterface $entity)
     }
 
     /**
+     * Create one object
+     *
+     * @param CrudEntityInterface $object Object to create
+     *
+     * @throws ValidationFailedException
+     * @return void
+     */
+    public function createOne(CrudEntityInterface $object)
+    {
+        $this->create($object, false);
+        if ($object instanceof ValidatableInterface) {
```

Comment:
Although I think this is a good suggestion, this will also further the breaking of backwards compatibility for anything that is currently using save. I will go ahead with my change. Please review those new changes carefully.